snap/snapcraft.yaml

```diff
@@ -38,6 +38,13 @@ build-packages:
   - sed
 
 parts:
+  cryptography-deps:
```

Comment:
this part is not strictly required, it might as well be part of a build-packages entry for where this is needed.